src/include/txn_inline.h

```diff
@@ -1223,6 +1223,20 @@ __wt_txn_read_upd_list_internal(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt,
 
             return (WT_PREPARE_CONFLICT);
         }
+
+        if (F_ISSET(upd, WT_UPDATE_RESTORED_FROM_DELTA) && upd->type == WT_UPDATE_STANDARD) {
+            /*
+             * If we see the first invisible update restored from delta, check the history store
```

Comment:
What's an "invisible update"? This could use a comment.

Reply:
The update that is not visible to the user so we need to read an older update.